fetch.bs

```diff
@@ -8105,11 +8105,11 @@ that RFC's normative processing requirements to be compatible with deployed cont
 
    <li><p>Remove trailing U+0020 SPACE <a>code points</a> from <var>mimeType</var>, if any.
 
-   <li><p>Remove the last U+003B (;) <a>code point</a> from <var>mimeType</var>.
+   <li><p>Remove the last U+003B (;) from <var>mimeType</var>.
   </ol>
 
- <li><p>If <var>mimeType</var> starts with U+003B (;), then prepend "<code>text/plain</code>"
- to <var>mimeType</var>.
+ <li><p>If <var>mimeType</var> <a for=string>starts with</a> U+003B (;), then prepend
```

Comment:
Technically this also should be `"<code>;</code>"` since it takes a string as input.